src/pages/layout/Search/SearchInput.tsx

```diff
@@ -1,10 +1,28 @@
 import React from "react";
-import {FormControl, InputAdornment, TextField} from "@mui/material";
+import {
+  FormControl,
+  InputAdornment,
+  TextField,
+  StandardTextFieldProps,
+} from "@mui/material";
 import SearchIcon from "@mui/icons-material/Search";
 
-export default function SearchInput({...params}) {
+interface SearchInputProps extends StandardTextFieldProps {
+  onSubmitSearch: () => void;
+}
+
+export default function SearchInput({
+  onSubmitSearch,
+  ...params
+}: SearchInputProps) {
   return (
-    <FormControl sx={{width: "100%"}}>
+    <form
```

Comment:
Isn't there a parent component that has a form? An input component should not be creating a form.

Reply:
Unfortunately I couldn't find a parent component that has a form in MUI.